docs/api.rst

```diff
@@ -2270,18 +2270,19 @@ Editing custom ethereum tokens
       Host: localhost:5042
       Content-Type: application/json;charset=UTF-8
 
-      {"token": {
-          "address": "0x1169C72f36A843cD3a3713a76019FAB9503B2807",
-          "decimals": 5,
-          "name": "foo",
-          "symbol": "FTK",
-          "started": 1614636432,
-          "swapped_for": "SCP,
-          "coingecko": "foo-coin",
-          "cryptocompare": "FOO",
-          "protocol": "aave",
-          "underlying_tokens": None
```

Comment:
wasn't the problem here that it was `None` and not `null`?

Reply:
Yes but then I changed it to null. And null was giving me this exception that I linked in discord

```
  File "/home/kelsos/.virtualenvs/rotki/lib/python3.7/site-packages/sphinxcontrib/httpexample/builders.py", line 213, in astify_json_obj
    raise Exception('Cannot astify {0:s}'.format(str(obj)))
Exception: Cannot astify None
```

Reply:
Yes but then I changed it to null. And null was giving me this exception that I linked in discord

```
  File "/home/kelsos/.virtualenvs/rotki/lib/python3.7/site-packages/sphinxcontrib/httpexample/builders.py", line 213, in astify_json_obj
    raise Exception('Cannot astify {0:s}'.format(str(obj)))
Exception: Cannot astify None
```